pkg/operator/reporters/incoming/incoming_reporter.go

```diff
@@ -0,0 +1,128 @@
+package incoming
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/eparis/bugzilla"
+
+	"github.com/mfojtik/bugzilla-operator/pkg/cache"
+	"github.com/mfojtik/bugzilla-operator/pkg/operator/config"
+	"github.com/mfojtik/bugzilla-operator/pkg/operator/controller"
+
+	"github.com/openshift/library-go/pkg/controller/factory"
+	"github.com/openshift/library-go/pkg/operator/events"
+)
+
+type IncomingReporter struct {
+	controller.ControllerContext
+	config     config.OperatorConfig
+	components []string
+}
+
+func (c *IncomingReporter) sync(ctx context.Context, syncContext factory.SyncContext) error {
+	client := c.NewBugzillaClient(ctx)
+	slackClient := c.SlackClient(ctx)
+
+	channelReport, assigneeReport, err := Report(ctx, client, syncContext.Recorder(), &c.config, c.components)
+	if err != nil {
+		return err
+	}
+	if len(assigneeReport) == 0 {
+		return nil
+	}
+
+	for assignee, bugs := range assigneeReport {
+		message := fmt.Sprintf("%s\n\n> Please set severity/priority on the bug(s) above and assign to a team member.\n", strings.Join(bugs.reports, "\n"))
+		if err := slackClient.MessageEmail(assignee, message); err != nil {
```

Comment:
Message*Email* ?

Reply:
assignee in BZ means email :-)

Reply:
iow. the slack client take email, convert it to slack user id and send message :-)

Reply:
ok. Misnamed....